SPECS/mstflint/mstflint.spec

```diff
@@ -29,9 +33,17 @@ Requires:       python3
 This package contains firmware update tool, vpd dump and register dump tools
 for network adapters based on Mellanox Technologies chips.
 
+%package kernel
+Summary:        mstflint Kernel Modules
+Group:          System Environment/Kernel
+Requires:       kernel >= 5.15.0 and kernel < 5.16.0
+Requires:       kernel-headers >= 5.15.0 and kernel-headers < 5.16.0
```

Comment:
Also, a similar request to use macros for upper and lower bounds for the versions of the kernel packages.